dll/win32/msv1_0/msv1_0.c

```diff
@@ -1244,29 +1247,36 @@ LsaApLogonUserEx2(IN PLSA_CLIENT_REQUEST ClientRequest,
         /* Fix-up pointers in the authentication info */
         PtrOffset = (ULONG_PTR)ProtocolSubmitBuffer - (ULONG_PTR)ClientBufferBase;
 
-        Status = RtlValidateUnicodeString(0, &LogonInfo->LogonDomainName);
+        /* We have to fix the string pointers.
+         * Changing the original pointer will make lsass unstable.
```

Comment:
But do we know why / the actual reason that triggers the instability?

Reply:
No. After some investigation i think its not needed. Sorry.
This fix cames from sspi-bringup. I found some other things that i've fixed in msv1_0.dll. Maybe the pointer thing does only "improve" stability in any way but not "really" fix something.

If you wish i can close the PR or revert this change. If i revert the only change is to improve some TRACEs.

Reply:
In the description of the PR you claim that it makes lsass unstable on XP (and 2003), so I don't know how you tested that (did you replace & use our msv1_0.dll on windows?)

Reply:
Yes, i've replaced msv1_0.dll on windows (xp and 2003). The patch came from my sspi-bringup. It was the first patch to LogonUserEx2. I've reverted this change im current sspi-bringup brach and no differnce (2003 works / xp failed with the same error as before).

Reply:
BTW: Current sspi-bringup msv1_0.dll is able too boot win2003+xp, win2003 can login (with and without password). And it seems to be stable.